examples/KitchenSink/src/LINEBot/KitchenSink/EventHandler/MessageHandler/TextMessageHandler.php

```diff
@@ -174,6 +176,40 @@ public function handle()
                 );
                 $this->bot->replyMessage($replyToken, $imagemapMessageBuilder);
                 break;
+            case 'imagemap video':
```

Comment:
`quickReply` is snake case. So this also should be in the same manner?

Reply:
change it to camel case!